pkg/extensions/search/common/common.go

```diff
@@ -215,6 +223,11 @@ func GetAnnotations(annotations, labels map[string]string) ImageAnnotations {
 		vendor = GetVendor(labels)
 	}
 
+	logo := GetLogo(annotations)
+	if logo == "" {
+		logo = GetLogo(annotations)
```

Comment:
```suggestion
		logo = GetLogo(labels)
```

If we want to support labels. I think we can. Otherwise the if is not needed.

Reply:
Done, changed.